src/Polly/CircuitBreaker/AsyncCircuitBreakerTResultSyntax.cs

```diff
@@ -18,55 +18,57 @@ private static void ValidateStaticDurationIsNonNegative(TimeSpan duration)
 
         /// <summary>
         /// <para> Builds a <see cref="AsyncPolicy{TResult}"/> that will function like a Circuit Breaker.</para>
-        /// <para>The circuit will break if <paramref name="exceptionsAllowedBeforeBreaking"/>
-        /// exceptions or results that are handled by this policy are encountered consecutively. </para>
+        /// <para>
+        /// The circuit will break if the number of consecutive actions resulting in a failure exceeds <paramref name="failureThreshold"/>.
+        /// </para>
         /// <para>The circuit will stay broken for the <paramref name="durationOfBreak"/>. Any attempt to execute this policy
         /// while the circuit is broken, will immediately throw a <see cref="BrokenCircuitException"/> containing the exception or result 
         /// that broke the circuit.
         /// </para>
-        /// <para>If the first action after the break duration period results in a handled exception or result, the circuit will break
-        /// again for another <paramref name="durationOfBreak"/>; if no exception or handled result is encountered, the circuit will reset.
+        /// <para>If the first action after the break duration period results in a failure, the circuit will break
+        /// again for another <paramref name="durationOfBreak"/>; if no failure is encountered, the circuit will reset.
         /// </para>
         /// </summary>
         /// <param name="policyBuilder">The policy builder.</param>
-        /// <param name="exceptionsAllowedBeforeBreaking">The number of exceptions or handled results that are allowed before opening the circuit.</param>
+        /// <param name="failureThreshold">The number of failures that are allowed before opening the circuit.</param>
         /// <param name="durationOfBreak">The duration the circuit will stay open before resetting.</param>
         /// <returns>The policy instance.</returns>
         /// <remarks>(see "Release It!" by Michael T. Nygard fi)</remarks>
-        /// <exception cref="ArgumentOutOfRangeException">exceptionsAllowedBeforeBreaking;Value must be greater than zero.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">failureThreshold;Value must be greater than zero.</exception>
         /// <exception cref="ArgumentOutOfRangeException">durationOfBreak;Value must be greater than zero</exception>
-        public static AsyncCircuitBreakerPolicy<TResult> CircuitBreakerAsync<TResult>(this PolicyBuilder<TResult> policyBuilder, int exceptionsAllowedBeforeBreaking, TimeSpan durationOfBreak)
+        public static AsyncCircuitBreakerPolicy<TResult> CircuitBreakerAsync<TResult>(this PolicyBuilder<TResult> policyBuilder, int failureThreshold, TimeSpan durationOfBreak)
```

Comment:
I do like these parameter name changes, but technically, they are breaking the public API (users may have quoted the existing parameter names in the calls).  That would mean we would push the release of this feature into the forthcoming v8.0.0 rather than a v7.3.0.

If speed of release is important to you for the feature, you might want to revert the parameter name changes to avoid the breaking change.

Reply:
Ah, shucks :)

Yup, I'll go and revert those and then have this change in a separate PR to go on afterwards.